### PR TITLE
switch links to main branch

### PR DESCRIPTION
**Pull request by ramshazar**
- I switched the git links to the new default branch "main"
- If the link also pointed to sig-security I switched it to tag-security

### CODE REVIEW

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md [hunk 1]
```diff
@@ -611,7 +611,7 @@ _图四_
 
 ## 角色和用例
 
-重点是安全、保护、检测和尽可能的自动响应。它不一定是独立的开发工具，而是透明地集成到开发流程中的安全工具，使得开发过程中就能够执行安全策略、快速反馈和直接补救。有关云原生安全用例的具体信息，请参考 [SIG-Security 的用例列表](https://github.com/cncf/sig-security/blob/master/usecases.md)。
+重点是安全、保护、检测和尽可能的自动响应。它不一定是独立的开发工具，而是透明地集成到开发流程中的安全工具，使得开发过程中就能够执行安全策略、快速反馈和直接补救。有关云原生安全用例的具体信息，请参考 [SIG-Security 的用例列表](https://github.com/cncf/tag-security/blob/main/usecases.md)。
```
Comment (ramshazar) [inline security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:614]: This is a 404. But It also was a 404 before the change.

File: .github/ISSUE_TEMPLATE/joint-review.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ Security Provider: yes/no (e.g. Is the primary function of the project to suppor
    - [ ] Project security lead
    - [ ] Lead security reviewer
    - [ ] 1 or more additional reviewer(s)
-   - [ ] Every reviewer has read [security reviewer guidelines](https://github.com/cncf/sig-security/blob/master/assessments/guide/security-reviewer.md) and stated declaration of conflict
+   - [ ] Every reviewer has read [security reviewer guidelines](https://github.com/cncf/tag-security/blob/main/assessments/guide/security-reviewer.md) and stated declaration of conflict
    - [ ] Sign off by 2 chairs on reviewer conflicts
 - [ ] Create slack channel (e.g. #sec-assess-projectname)
-- [ ] Project lead provides draft document - see [outline](https://github.com/cncf/sig-security/blob/master/assessments/guide/outline.md)
+- [ ] Project lead provides draft document - see [outline](https://github.com/cncf/tag-security/blob/main/assessments/guide/outline.md)
```
Comment (ramshazar) [inline .github/ISSUE_TEMPLATE/joint-review.md:29]: This is a 404. But It also was a 404 before the change.

Comment (ultrasaurus) [reply .github/ISSUE_TEMPLATE/joint-review.md:29]: @lumjjb @TheFoxAtWork should this be https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md ?

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md (hunk 1 above)
Comment (ultrasaurus) [reply security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:614]: this link should be https://github.com/cncf/tag-security/tree/main/usecase-personas

**Review by ultrasaurus — APPROVED** [review]
LGTM -- for the 404s, pls write up as issues, if not fixed, before merging, so we don't lose track of 'em

thanks so much!

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md (hunk 1 above)
Comment (ultrasaurus) [reply security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:614]: see issue https://github.com/cncf/tag-security/issues/649

File: .github/ISSUE_TEMPLATE/joint-review.md (hunk 2 above)
Comment (ultrasaurus) [reply .github/ISSUE_TEMPLATE/joint-review.md:29]: I didn't see the file, so I don't think the link I provided is correct.  Wrote up issue: https://github.com/cncf/tag-security/issues/650

**Comment by ultrasaurus** [comment]
I'm going to go ahead and merge this with just my review, because this seems to fit the criteria of [simple change](https://github.com/cncf/tag-security/blob/main/CONTRIBUTING.md#simple-changes) and leaving this as is in the repo could cause confusion.

Thank you @ramshazar for your contribution!

**Pull request merged**